docs/markdown/releasenotes.md

```diff
@@ -29,38 +35,48 @@
 ## Version 7.8.0
 
 ### New features
+
 * Added support for the pnpm package manager.
 * Added property detect.project.group.name for setting the Project Group.
 * [solution_name] now falls back to using a previously-downloaded Docker Inspector, Project Inspector, and/or NuGet Inspector when https://sig-repo.synopsys.com is unreachable.
 
 ## Version 7.7.0
 
 ### New features
+
 * Added support for uploading rapid scan config file when a file named '.bd-rapid-scan.yaml' is present in the source directory.
 * Added the property detect.project.inspector.arguments for providing additional arguments to the project inspector across all invocations.
 
 ### Resolved issues
+
 * (IDETECT-2808, IDETECT-2863) Resolved an issue where [solution_name] would incorrectly resolve relative paths when processing signature scan targets.
 * (IDETECT-2859) Resolved an issue where [solution_name] was using an outdated cookie spec when making a request, resulting in a warning message.
 
 ## Version 7.6.0
 
 ### New features
-* Added the property [detect.follow.symbolic.links](properties/configuration/general.md#follow-symbolic-links) which can be used to enable [solution_name] to follow symbolic links when searching directories for detectors, when creating exclusions for signature scan, and when creating binary scan targets.
+
+* Added the property [detect.follow.symbolic.links](properties/configuration/general.md#follow-symbolic-links) which can be used to enable [solution_name] to follow symbolic links when searching directories for detectors, when creating
```

Comment:
Change to read: "When creating exclusions for signature scan or when creating binary scan targets, the new  property [detect.follow.symbolic.links](properties/configuration/general.md#follow-symbolic-links) enables [solution_name] to follow symbolic links when searching directories for detectors."